src/App.tsx

```diff
@@ -184,8 +191,31 @@ export default function App() {
                   <Redirect from="/updates" to="/covid-explained" />
                   {/* TODO(pablo): Route every article */}
                   <Route from="/deep-dives" component={DeepDivesRedirect} />
+                  {/*
+                    If path is /covid-risk-levels-metrics,
+                    redirect to /covid-community-level-metrics and
+                    scroll to appropriate section if path has a hash.
+
+                    More info on render prop: https://v5.reactrouter.com/web/api/Route/render-func
+                  */}
                   <Route
                     path="/covid-risk-levels-metrics"
+                    render={props => {
+                      if (
+                        props.location.hash &&
+                        metricExplainerHashes.includes(props.location.hash)
+                      ) {
+                        return (
+                          <Redirect
+                            to={`covid-community-level-metrics${props.location.hash}`}
+                          />
+                        );
+                      }
+                      return <Redirect to="/covid-community-level-metrics" />;
+                    }}
+                  />
```

Comment:
Let me know if there's a better solution here. At first I tried:
```js
                  <Redirect
                    from="/covid-risk-levels-metrics#icu-capacity-used"
                    to="/covid-community-level-metrics#icu-capacity-used"
                  />
                  <Redirect
                    from="/covid-risk-levels-metrics#daily-new-cases"
                    to="/covid-community-level-metrics#daily-new-cases"
                  />
                  <Redirect
                    from="/covid-risk-levels-metrics#positive-test-rate"
                    to="/covid-community-level-metrics#positive-test-rate"
                  />
                  <Redirect
                    from="/covid-risk-levels-metrics#infection-rate"
                    to="/covid-community-level-metrics#infection-rate"
                  />
```
And the redirect seems to disregard the hash (i.e. it redirects to the correct page but doesn't scroll to the appropriate section). Tried playing around with props like [exact](https://v5.reactrouter.com/web/api/Redirect/exact-bool) and [strict](https://v5.reactrouter.com/web/api/Redirect/strict-bool) too. Found one possible solution in [this thread](https://stackoverflow.com/questions/45864594/react-router-redirect-hash-link).

Reply:
Hrm.  Is there a reason we have to actually validate that the hash is in `metricExplainerHashes`?  Why not just always preserve the hash with 

```
                          <Redirect
                            to={`covid-community-level-metrics${props.location.hash}`}
                          />
```

or whatever?

Reply:
We could definitely do that. I just put this check there to be explicit about which hashes _should_ be valid ones. That said, the behaviour should be the same either way. I'll keep it simple and stick with your solution.